tmt/steps/__init__.py

```diff
@@ -804,6 +805,9 @@ def go(self) -> None:
 class Login(Action):
     """ Log into the guest """
 
+    # Change typing of inherited attr
```

Comment:
Please, add a `TODO: remove when Step becomes Generic (#1372)` for future generations of engineers who would address this issue.

Reply:
Done